Specs/Renderer/DrawSpec.js

```diff
@@ -1172,4 +1233,78 @@ defineSuite([
             });
         }).toThrowDeveloperError();
     });
-}, 'WebGL');
\ No newline at end of file
+
+    it('throws if instanceCount is less than one', function() {
+        var vs = 'attribute vec4 position; void main() { gl_PointSize = 1.0; gl_Position = position; }';
+        var fs = 'void main() { gl_FragColor = vec4(1.0); }';
+        sp = ShaderProgram.fromCache({
+            context : context,
+            vertexShaderSource : vs,
+            fragmentShaderSource : fs
+        });
+
+        va = new VertexArray({
+            context : context,
+            attributes : [{
+                index : sp.vertexAttributes.position.index,
+                vertexBuffer : Buffer.createVertexBuffer({
+                    context : context,
+                    typedArray : new Float32Array([0, 0, 0, 1]),
+                    usage : BufferUsage.STATIC_DRAW
+                }),
+                componentsPerAttribute : 4
+            }]
+        });
+
+        var command = new DrawCommand({
+            primitiveType : PrimitiveType.POINTS,
+            shaderProgram : sp,
+            vertexArray : va,
+            instanceCount : -1
+        });
+
+        expect(function() {
+            command.execute(context);
+        }).toThrowDeveloperError();
+    });
+
+    it('throws when instanceCount is greater than one and the instanced arrays extension is not supported', function() {
+        // disable extension
+        var instancedArrays = context._instancedArrays;
+        context._instancedArrays = undefined;
```

Comment:
See here for an example: https://github.com/AnalyticalGraphicsInc/cesium/blob/f846b3a999599a33e738098458fd877110b44e85/Specs/Scene/CameraSpec.js#L2133